src/main/java/uk/gov/mca/beacons/service/beacons/BeaconsResponseFactory.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.mca.beacons.service.beacons;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import uk.gov.mca.beacons.service.dto.BeaconDTO;
+import uk.gov.mca.beacons.service.dto.BeaconPersonDTO;
+import uk.gov.mca.beacons.service.dto.BeaconUseDTO;
+import uk.gov.mca.beacons.service.dto.DomainDTO;
+import uk.gov.mca.beacons.service.dto.RelationshipDTO;
+import uk.gov.mca.beacons.service.dto.WrapperDTO;
+import uk.gov.mca.beacons.service.model.Beacon;
+
+public class BeaconsResponseFactory {
+
+  public static WrapperDTO<BeaconDTO> buildDTO(Beacon beacon) {
+    List<DomainDTO> useDTOs = getUseDTOs(beacon);
+    BeaconPersonDTO ownerDTO = getOwnerDTO(beacon);
+    List<DomainDTO> emergencyContactDTOs = getEmergencyContactDTOs(beacon);
```

Comment:
Can all be made `final` , i.e: `final List<DomainDTO>` etc for these and on line 20